### ISSUE TITLE
Migration Generator on Rails 6

### ISSUE THREAD

**Issue by wilpar** [ryanto/acts_as_votable#195]
Attempting to install 0.12 on a 6.0.0rc1 project

    Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:
     class ActsAsVotableMigration < ActiveRecord::Migration[4.2]

I guess 4.2 is too old for 6.0 now - any known problems with me tagging it as a 6.0 migration?

```
== 20190531163739 ActsAsVotableMigration: migrating ===========================
-- create_table(:votes)
   -> 0.0116s
-- add_index(:votes, [:voter_id, :voter_type, :vote_scope])
   -> 0.0021s
-- add_index(:votes, [:votable_id, :votable_type, :vote_scope])
   -> 0.0024s
== 20190531163739 ActsAsVotableMigration: migrated (0.0170s) ==================
```

**Comment by ryanto** [comment]
Yup, should be fine! If you PR we can release a new version.

**Comment by wilpar** [comment]
it's been a while, will try to do so.

**Comment by BillyParadise** [comment]
took longer to fight with ssh keys than to make the pr ;)

**Comment by theasteve** [comment]
is it still open?

**Comment by ryanto** [comment]
Thanks @BillyParadise ! 

Could you look at why the tests are failing? We would need those to pass before merging.

Again thanks so much for addressing this!

**Comment by BillyParadise** [comment]
They're failing because there isn't a profile for rails 6 in the CI tool config.  Not something I'm familiar with generating - I'll have to put on my learning hat to figure it out.  In the meantime just putting a [6.0] manually into the migration works just fine.

**Comment by murny** [comment]
@ryanto @BillyParadise I created this PR to fix the broken build https://github.com/ryanto/acts_as_votable/pull/198

Once merged you should be able to just pull latest from master and your branch should succeed without issues 👍

**Comment by ryanto** [comment]
Awesome, thank you both @BillyParadise and @murny ! Merged in this PR: https://github.com/ryanto/acts_as_votable/pull/202

**Comment by ryanto** [comment]
Alright everyone thanks for being so patient! This has been released as version `0.12.1`
